modules/ROOT/pages/dw-arrays-functions-countby.adoc

```diff
@@ -8,7 +8,7 @@ endif::[]
 [[countby1]]
 == countBy&#40;Array<T&#62;, &#40;T&#41; &#45;&#62; Boolean&#41;: Number
 
-Counts the elements in an array that match the results of a function.
+Counts the elements in an array that return `true` when the matching function is applied to the value of the element.
 
 
 === Parameters
```

Comment:
I know this is legacy stuff, but I saw it and wondered about Parameters and Example being at the same level.
I wonder if they should be combined?
Or I wonder if the titles should be "<blah> Parameters" and "<blah> Example"?